mm2src/adex_cli/Cargo.toml

```diff
@@ -22,7 +22,7 @@ sysinfo = "0.28"
 tiny-bip39 = "0.8.0"
 tokio = { version = "1.20", features = [ "macros" ] }
 
-[target.'cfg(all(not(target_arch = "wasm32"), not(target_os = "macos"), not(windows)))'.dependencies]
+[target.'cfg(all(target_family = "unix", not(target_os = "macos")))'.dependencies]
```

Comment:
is `target_os = "linux"` doesn't do the work?

Reply:
It should work on other unix, excluding macos

Reply:
I see, thank you